appinventor/appengine/src/com/google/appinventor/client/style/neo/DesignToolbarNeo.ui.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2009-2011 Google, All Rights reserved -->
+<!-- Copyright 2011-2023 MIT, All rights reserved -->
```

Comment:
Since the neo UI is our own, we don't need to include the Google copyright here.